microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -137,6 +294,24 @@ Follow these steps to get the product ID:
 
 <!-- ---------------------------------- -->
 #### Sample request
+<!-- for endpoint 1 -->
+
+To upload a package to update an existing submission, use the `curl` command at a command prompt as follows:
+
+##### [v1.1](#tab/v1-1)
+
+```console
+> curl \
+-H "Authorization: ApiKey $ApiKey" \
+-H "X-ClientID: $ClientID" \
+-H "Content-Type: application/zip" \
+-X POST \
+-T $FILE_NAME \
+-v \
+https://api.addons.microsoftedge.microsoft.com/v1/products/$productID/submissions/draft/package
```

Comment:
pattern 2 of /v1/ question
```suggestion
https://api.addons.microsoftedge.microsoft.com/v1-1/products/$productID/submissions/draft/package
```
in v1.1 tabs in using .md, change /v1/ to /v1-1/ ?
@yvsubhash @bhuvanapriyap

Reply:
Ans: URL continues to be /v1/, that's why we changed from "v2" to "v1.1".  
url is same for v1.1 as for v1